package.json

```diff
@@ -43,7 +43,7 @@
         "@wojtekmaj/enzyme-adapter-react-17": "^0.6.1",
         "autoprefixer": "^9.8.6",
         "babel-eslint": "^10.0.2",
-        "babel-jest": "^26.3.0",
+        "babel-jest": "^27.3.1",
```

Comment:
unfortunately, it looks like the new [`jest-circus` runner](https://jestjs.io/blog/2021/05/25/jest-27#flipping-defaults) fails when comparing a **normal array** with an **observable array** now. this leads to a lot of errors like this when running the tests:
```
 FAIL  src/Sulu/Bundle/AdminBundle/Resources/js/containers/FieldBlocks/tests/FieldBlocks.test.js

  ● Should pass new value to the BlockCollection if value prop is updated

    expect(received).toEqual(expected) // deep equality

    Expected: [{"text": "One", "type": "default"}]
    Received: serializes to the same string
```

Reply:
i suspect that this problem could be solved by updating to a new version of mobx. but unfortunately we are stuck at version 4 until the next major release (see #6343) 😕